tests/Unit/OutputFormatTest.php

```diff
@@ -684,4 +685,181 @@ public function setIndentationLevelProvidesFluentInterface(): void
+
+    /**
+     * @test
+     */
+    public function indentWithSpacesByDefaultSetsIndentationToTwoSpaces(): void
+    {
+        $this->subject->indentWithSpaces();
+
+        self::assertSame('  ', $this->subject->getIndentation());
+    }
+
+    /**
+     * @return array<string, array{0: int<0, max>, 1: string}>
+     */
+    public static function provideSpaceIndentation(): array
+    {
+        return [
+            'zero spaces' => [0, ''],
+            'one space' => [1, ' '],
+            'two spaces' => [2, '  '],
+            'three spaces' => [3, '   '],
+            'four spaces' => [4, '    '],
+        ];
+    }
+
+    /**
+     * @test
+     * @dataProvider provideSpaceIndentation
+     */
+    public function indentWithSpacesSetsIndentationToTheProvidedNumberOfSpaces(
+        int $numberOfSpaces,
+        string $expectedIndentation
+    ): void {
+        $this->subject->indentWithSpaces($numberOfSpaces);
+
+        self::assertSame($expectedIndentation, $this->subject->getIndentation());
+    }
+
+    /**
+     * @test
+     */
+    public function indentWithSpacesProvidesFluentInterface(): void
+    {
+        self::assertSame($this->subject, $this->subject->indentWithSpaces());
+    }
+
+    /**
+     * @test
+     */
+    public function nextLevelReturnsOutputFormatInstance(): void
+    {
+        self::assertInstanceOf(OutputFormat::class, $this->subject->nextLevel());
+    }
+
+    /**
+     * @test
+     */
+    public function nextLevelReturnsDifferentInstance(): void
+    {
+        self::assertNotSame($this->subject, $this->subject->nextLevel());
+    }
+
+    /**
+     * @test
+     */
+    public function nextLevelReturnsInstanceWithIndentationLevelIncreasedByOne(): void
+    {
+        $originalIndentationLevel = 2;
+        $this->subject->setIndentationLevel($originalIndentationLevel);
+
+        self::assertSame($originalIndentationLevel + 1, $this->subject->nextLevel()->getIndentationLevel());
+    }
+
+    /**
+     * @test
+     */
+    public function beLenientSetsIgnoreExceptionsToTrue(): void
+    {
+        $this->subject->setIgnoreExceptions(false);
+
+        $this->subject->beLenient();
+
+        self::assertTrue($this->subject->getIgnoreExceptions());
+    }
+
+    /**
+     * @test
+     */
+    public function getFormatterReturnsOutputFormatterInstance(): void
+    {
+        self::assertInstanceOf(OutputFormatter::class, $this->subject->getFormatter());
+    }
+
+    /**
+     * @test
+     */
+    public function getFormatterCalledTwoTimesReturnsSameInstance(): void
+    {
+        $firstCallResult = $this->subject->getFormatter();
+        $secondCallResult = $this->subject->getFormatter();
+
+        self::assertSame($firstCallResult, $secondCallResult);
+    }
+
+    /**
+     * @test
+     */
+    public function levelReturnsIndentationLevel(): void
+    {
+        $value = 4;
+        $this->subject->setIndentationLevel($value);
+
+        self::assertSame($value, $this->subject->level());
```

Comment:
Should this be calling `getIndentationLevel()`, since `level()` was deprecated in #870.

Reply:
No, this test is specific to the (deprecated) method. I'll remove the test together with the method.

(`setIndentationLevel` itself has a dedicated test.)

Reply:
Ah, yes, I overlooked the test method name.